botocore/httpchecksum.py

```diff
@@ -32,10 +32,7 @@
     MissingDependencyException,
 )
 from botocore.response import StreamingBody
-from botocore.utils import (
-    conditionally_calculate_md5,
-    determine_content_length,
-)
+from botocore.utils import _has_checksum_header, determine_content_length
```

Comment:
If we're going to start importing it, then we'll need to make it public since it's used cross module. We can leave a note that it's not intended for external usage.

Reply:
Updated in 75db0fe6792083943d1af499275fbcfea77b3666